engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html

```diff
@@ -118,6 +118,11 @@ <h6 translate>
                 ng-model="repository.download_policy"
                 ng-options="id as name for (id, name) in downloadPolicies">
         </select>
+        <p class="help-block" translate>
+          For on demand synchronization, only the metadata is downloaded during sync and packages are fetched and stored on the filesystem when clients request them.
+          For background syncs, a background task will download all packages after the initial sync.
+          The immediate option is what Katello by default provides where all metadata and packages are downloaded during sync.
```

Comment:
I would use the same capitalization as the drop down:  "On Demand", "Background" and "Immediate".

Also the default is not "On Demand" currently.  You might just say something like "The Immediate option will download all metadata and packages immediately during the sync".  Since this is the UI, the default will be auto-selected (and can be changed by the user).